src/index.html

```diff
@@ -17,16 +17,20 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form method="post">
+      <input
+        class="input big-input"
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+      />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <input
+        class="input small-input"
+        type="text"
+        data-qa="small"
+        placeholder="Try “Los Angeles“"
```

Comment:
Keep your attributes correctly formatted. If the HTML element has long attribute values or the number of attributes is more than 2, start each one, including the first, on a new line with 2-space indentation related to the tag.